django_squash/management/commands/lib/autodetector.py

```diff
@@ -133,9 +161,11 @@ def add_non_elidables(self, original, loader, changes):
                 for operation in all_custom_operations(migration.operations, unique_names):
                     if isinstance(operation, migration_module.RunPython):
                         operation.code = copy_func(operation.code)
+                        operation.code.__original_module__ = operation.code.__module__
```

Comment:
You can ignore this suggestion, but when adding our properties to someone else's object, you want to name the property in a way that clearly identifies that it is ours and the creators of that object will never use our property name. Instead of naming it dunder style, which the original authors might use in the future, I would prefix props we add to their objects with "squasher_". Example: "squasher_original_module".

Reply:
This is the kind of thing typehint/pyright will absolutely hate me for, but i can't figure doing any other way. `code` is the function object itself. Unless python in the future starts to use `__original_module__` we _should_ be ok.